spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,15 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        in_nodes = assigns(:order_details)
+                   .where_values.select { |v| v.is_a?(Arel::Nodes::In) }
+        expect(in_nodes).to satisfy { |nodes|
+          nodes.any? do |node|
+            node.left.name == :user_id &&
+              node.left.relation.name.casecmp("account_users").zero? &&
+              node.right.map(&:val) == ["3", "4"]
+          end
```

Comment:
This is a rubocop thing to add `casecmp`, there's a clearer `casecmp?` method, but I think that's Ruby 2.4.